dns-root-hints/verisign-grs-nstld-key.asc

```diff
@@ -0,0 +1,50 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
This is also the same approach as Alpine https://git.alpinelinux.org/aports/plain/main/dns-root-hints/verisign-grs-nstld-key.asc